diff -aurN cups-1.7.5.orig/config-scripts/cups-directories.m4 cups-1.7.5/config-scripts/cups-directories.m4
--- cups-1.7.5.orig/config-scripts/cups-directories.m4	2014-09-28 20:26:02.962652729 +0200
+++ cups-1.7.5/config-scripts/cups-directories.m4	2014-09-28 20:26:22.670066676 +0200
@@ -420,7 +420,7 @@
 			;;
 		*)
 			# All others
-			CUPS_STATEDIR="$localstatedir/run/cups"
+			CUPS_STATEDIR="/run/cups"
 			;;
 	esac])
 AC_DEFINE_UNQUOTED(CUPS_STATEDIR, "$CUPS_STATEDIR")
